dev/com.ibm.ws.jaxws.2.3.common/src/com/ibm/ws/jaxws/client/LibertyWebServiceClientInInterceptor.java

```diff
@@ -87,17 +78,32 @@ public void handleMessage(Message message) throws Fault {
                 
             }
 
-            
+            // Liberty change begin
+            // if messageServiceName != null, try to get the values from configuration using it
+            if (messageServiceName != null) {
```

Comment:
Since this is within the first `if (messageServiceName != null)` you don't need to check it again.